platform/ios/PolyPodApp/PolyPodUITests/SmokeTest.swift

```diff
@@ -0,0 +1,93 @@
+// Please remove this line and the empty one after it
+
+import XCTest
+
+class SmokeTest: XCTestCase {
+    private let app = XCUIApplication()
+    
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        continueAfterFailure = false
+    }
+
+    func testOpenFeaturesNoErrorPopUp() throws {
+        app.launchWithArgs()
+        assertHomeScreenIsShown()
+        
+        let tile_identifiers = ["big_card_view", "medium_card_view", "small_card_view"]
```

Comment:
Could there be just one identifier for all of the cards?

Reply:
In #1144, I'm adding a `$name_feature` accessibility identifier to all the development features. Maybe it would work to add that identifier to _every_ feature card, and then just iterate over all those?

Alternative ideas (probably more what Ghenadie meant):

1. Matching any identifier that ends in `_card_view`
2. Giving each of those cards the same identifier, e.g. just `card_view` or `feature_card_view`

Reply:
Also nit: I think someone did too much Rust :D Camel case and all that.

Reply:
Yes, one identifier should work. That's a good idea. 

Yeah, maybe too much rust 😆